frontend/src/players/players-list/players-list.tsx

```diff
@@ -53,86 +73,53 @@ const PlayerPositionContainer = styled.div`
 `;
 
 const AddPlayerButtonContainer = styled.div`
-  flex-basis: 100%;
-  flex-grow: 1;
-  margin-top: 10px;
+  padding-top: 5px;
+  margin: auto auto 0 auto;
 `;
 
 const AddPlayerLink = styled(NavLink)``;
 
 type Props = {
   team: Team;
-  onFormSubmit: (team: Team) => void;
+  deletePlayer: (teamId: number, playerId: number) => void;
 };
 
-const tempConfig = [
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  // { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  // { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  // { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  // { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  // { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-  // { name: "essa1", surname: "essa2", position: "essa3", id: "1" },
-];
-
-export const PlayersList = ({ team, onFormSubmit }: Props): JSX.Element => {
-  const [players, setPlayers] = useState<Player[]>([]);
-
-  const api = playersAPI();
+export const PlayersList = ({ team, deletePlayer }: Props): JSX.Element => {
+  const players = team.players ? team.players : [];
 
   const addPlayerUrl = `/teams/${team.id}/player/create`;
 
-  useEffect(() => {
-    setPlayers(team.players?.length ? team.players : []);
-  }, [team.players]);
-
-  const deletePlayer = (playerId: number) => {
-    api.deletePlayer(team.id, playerId);
-    setPlayers(players.filter((player) => player.id !== playerId));
+  const onDeletePlayer = (playerId: number) => {
```

Comment:
It is meaningless to wrap `deletePlayer` in another function. Just refactor parent function `deletePlayer` that it wouldnt need teamId. look at how it is done right now. You have API call deletePlayer which is wrapped in another function, and then you passed it to children and second time wrapped it in function.